src/plat/bcm2712/overlay-rpi5-2gb.dts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright 2025, UNSW
+ * SPDX-License-Identifier: GPL-2.0-only
+ */
+
+/ {
+    /delete-node/ memory;
+
+    /* Memory for the 2GB RAM variant of the RPi5B, see overlay-pi5.dts for an
+     * explanation of the memory ranges.
```

Comment:
There is no such explanation there and it should be here anyway?

Reply:
Fixed.